tails_files/install.sh

```diff
@@ -277,14 +277,6 @@ EOL
   fi
 }
 
-function configure_ansible_apt_persistence()
-{
-  # set ansible to auto-install
-  if ! grep -q 'ansible' "${tails_live_persistence}/live-additional-software.conf"; then
-    echo "ansible" >> "${tails_live_persistence}/live-additional-software.conf"
```

Comment:
> Do those need to be reinstalled after every boot, or is the virtualenv effectively fully self-contained once it is created?

It's the latter so far during testing, the virtualenv is contained and doesnt require the other apt dependencies.